### ISSUE TITLE
CLI - Failed to fetch tree from registry

### ISSUE THREAD

**Issue by TryBlock** [shadcn-ui/ui#1478]
I have a next.js app and I am trying to use "**shadcn-ui@latest add**" command in the terminal but I am getting the following error.

shadcn-ui@latest add dropdown-menu popover separator badge command select

FetchError: request to https://ui.shadcn.com/registry/styles/default/select.json failed, reason: read ECONNRESET
    at ClientRequest.<anonymous> (file:///D:/.pnpm-store/v3/tmp/dlx-5248/node_modules/.pnpm/node-fetch@3.3.2/node_modules/node-fetch/src/index.js:108:11)
    at ClientRequest.emit (node:events:513:28)
    at TLSSocket.socketErrorListener (node:_http_client:502:9)
    at TLSSocket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  type: 'system',
  errno: 'ECONNRESET',
  code: 'ECONNRESET',
  erroredSysCall: 'read'
}
Failed to fetch tree from registry.

**Comment by nelsonfrz** [comment]
I am experiencing the same issue :(

**Comment by wvsr** [comment]
I think this is your network issue, Did you try to extend the timeout of npm/yarn?

**Comment by zhihancn** [comment]
i am meeting the same issue.

my browser can request the https://ui.shadcn.com/registry/styles/default/select.json successfully.

but terminal cannot

**Comment by shadcn** [comment]
This issue has been automatically closed because it received no activity for a while. If you think it was closed by accident, please leave a comment. Thank you.

**Comment by wuqianrui** [comment]
My operating environment is Windows. I used Google to look up the IP address for `https://www.shadcn-vue.com/registry/styles/index.json` and found it to be `104.21.17.226` . and  I edited the hosts file on my Windows computer, adding the line:
```text
104.21.17.226 www.shadcn-vue.com
```
With this addition, the configuration takes effect locally for me; however, some Windows devices might require a restart for the changes in the hosts file to be applied.

Side note: As a Chinese person, dealing with these sorts of network issues can often be quite draining.
